hypothesis-python/tests/cover/test_sampled_from.py

```diff
@@ -51,3 +51,14 @@ def test_can_sample_enums(member):
 @checks_deprecated_behaviour
 def test_sampling_empty_is_deprecated():
     assert sampled_from([]).is_empty
+
+
+@fails_with(FailedHealthCheck)
+@given(sets(sampled_from(range(10)), min_size=11))
+def test_unsat_sets_of_samples(x):
+    assert False
+
+
+@given(sets(sampled_from(range(100)), min_size=100))
+def test_efficient_sets_of_samples(x):
+    assert x == set(range(100))
```

Comment:
These tests don't appear to be needed for coverage, and `test_efficient_sets_of_samples` is just slow enough (~2 seconds for me) that I could see moving them into a separate `tests/nocover/test_sampled_from.py`.